infra/gravity/testsuite.go

```diff
@@ -179,10 +179,9 @@ func (s *testSuite) getLogLink(testUID string) (string, error) {
 			"expandAll": []string{"false"},
 			"authuser":  []string{"1"},
 			"advancedFilter": []string{
-				fmt.Sprintf(`resource.type="project"
+				fmt.Sprintf(`severity>=INFO
 labels.__uuid__="%s"
-labels.__suite__="%s"
```

Comment:
Can you elaborate a bit - does this still work when this is not run from a GCP node?

Reply:
> does this still work when this is not run from a GCP node?

Yes it does. `labels.__suite__` and `labels.__uuid__` are sufficient unique (and indexed) fields to uniquely identify all logs from a robotest run.  We could add another label or a `logName` if we're concerned that something else may sink logs that collide (unlikely given a two UUIDs).

The difference is robotest was filtering on `resource.type="project"`.  However, when run from a gce node, `resource.type="gce_instance"`.  I haven't found a way to turn that off -- it seems to be metadata added on the receiving side instead of the logging client configuration.

There are example metadata in https://github.com/gravitational/robotest/issues/265.

Reply:
Ah, I think I misinterpreted the changeset (it appeared to me initially as if you were removing the `__suite__` attribute).